pre_award/assess/assessments/routes.py

```diff
@@ -1346,8 +1351,8 @@ def request_changes(application_id, sub_criteria_id, theme_id):
         "assessments/request_changes.html",
         form=form,
         question_choices=[
-            {"text": label, "value": value, "checked": value in (form.field_ids.data or [])}
-            for value, label in form.field_ids.choices
+            {"text": label, "value": value, "response": answer, "checked": value in (form.field_ids.data or [])}
```

Comment:
I think you can use directly `theme_answers_response` to build this dictionary for the template.

The tuple we had before had too many items, that we only really needed here: question and answer property.

So when we were looping we were doing the `_`, as we were not using them. I think other programming languages don't mind it. But in Python I don't see it often. I use it only when I am lazy. But I wanted to avoid it here.